tests/test_view.py

```diff
@@ -105,3 +149,50 @@ def test_view_share_updates_dictionary_not_overwrite():
     viewclass.share({'test2': 'test2'})
 
     assert viewclass.dictionary == {'test1': 'test1', 'test2': 'test2'}
+
+
+def test_view_cache():
+    """
+    Test for cache template each 5 min
+    """
+
+    container = App()
+
+    view = View(container)
+
+    container.bind('CacheConfig', cache)
+    container.bind('CacheDiskDriver', CacheDiskDriver)
+    container.bind('CacheManager', CacheManager(container))
+    container.bind('Application', container)
+    container.bind('Cache', container.make('CacheManager').driver('disk'))
+    container.bind('View', view.render)
+
+    view = container.make('View')
+
+    assert view(
+        'test_cache', {'test': 'test'}
+    ).cache_for(1, 'second').rendered_template == 'test'
+
+    assert open(glob.glob('bootstrap/cache/test_cache:*')[0]).read() == 'test'
+
+    time.sleep(2)
+
+    assert view(
+        'test_cache', {'test': 'macho'}
+    ).cache_for(5, 'seconds').rendered_template == 'macho'
+
+    time.sleep(5)
```

Comment:
Change this to `time.sleep(2)` to speed up tests. it should still work and make tests faster